Add missing DIY video guides (#360)
pupil-labs/pupil-docs: src/core/diy/README.md

## De-case Cameras
Take both webcams out of their casings. Follow the video guides.

- [Logitech C525/C512](https://vimeo.com/59844059)

> <iframe src="https://player.vimeo.com/video/59844059" width="640" height="360" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>

Reviewer: ```suggestion
```

Reviewer: remove video embed, just link to the video.

- [Microsoft HD-6000](https://vimeo.com/53005603)

> <iframe src="https://player.vimeo.com/video/53005603" width="640" height="360" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>

Reviewer: ```suggestion
```

Reviewer: remove iframe/embed - just use the link above.

> - Cut off the microphon

Reviewer: ```suggestion
- Cut off the microphone
```

- De-solder or break off the push button (Note: Some cameras don’t have this button.)
- De-solder the blue LED’s

> - solder on the IR-LED’s. Please take note of LED polarity!

Reviewer: ```suggestion
- Solder on the IR-LED’s. Please take note of LED polarity! [Video guide](https://youtu.be/O-FAXldfq94 "Solder IR LEDs Pupil DIY")
```

> <iframe width="640" height="360" src="https://www.youtube.com/embed/O-FAXldfq94" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

Reviewer: ```suggestion
```

## Camera Assembly
1. Attach the world camera onto the mount using 4 small screws, leftover from disassembly.
1. Clip the world camera clip onto the headset

> 1. Slide the eye-cam into the mount [video guide](https://www.youtube.com/watch?v=wkV9Ye7psP4)

Reviewer: ```suggestion
1. Slide the eye-cam into the mount [video guide](https://youtu.be/wkV9Ye7psP4 "Pupil DIY: Attach eye camera")
```
